sites/TR-FC1-ULAKBIM.yaml

```diff
@@ -28,3 +28,6 @@ vos:
 - auth:
     project_id: a59bd0a9ed754941962846c4a081d6ca
   name: vo.enes.org
+  - auth:
```

Comment:
```suggestion
- auth:
```

Reply:
Sorry, my bad. Seems I overlooked while writing it in. Thanks for spotting it.